dashboard/config/scripts/csp_u3_turtle_subgoal_q1_mc_2018_2019.multi

```diff
@@ -1,5 +1,4 @@
 name 'csp_U3_turtle_subgoal_q1_mc_2018_2019'
-editor_experiment ''
```

Comment:
Is this intentional?

Reply:
it was the result of resolving the merge from levelbuilder; not strictly intentional, but neither unexpected nor significant